query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseFunctionField.java

```diff
@@ -13,30 +13,66 @@
  **/
 package org.codice.ddf.admin.common.fields.base;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
+import java.util.stream.Stream;
 
 import org.codice.ddf.admin.api.DataType;
 import org.codice.ddf.admin.api.fields.FunctionField;
 import org.codice.ddf.admin.api.report.ErrorMessage;
 import org.codice.ddf.admin.api.report.FunctionReport;
 import org.codice.ddf.admin.api.report.Report;
 import org.codice.ddf.admin.common.report.FunctionReportImpl;
+import org.codice.ddf.admin.common.report.message.DefaultMessages;
 
 import com.google.common.collect.ImmutableList;
 
 public abstract class BaseFunctionField<T extends DataType> extends BaseField<Map<String, Object>, FunctionReport<T>> implements FunctionField<T> {
 
     private FunctionReportImpl<T> report;
 
+    private String description;
+
     public BaseFunctionField(String functionName, String description) {
         super(functionName, description);
+        this.description = description;
         report = new FunctionReportImpl<>();
     }
 
+    @Override
+    public String description() {
+        if (this.getErrorCodes() != null) {
+            return String.format("%s The possible errors are:    \r\n %s.",
```

Comment:
There's whitespace because we want to display the errors in a new line and since graphQL uses markdown, two spaces followed by /r/n means new line.